updates/version.yaml

```diff
@@ -49,3 +49,4 @@
 1.3.7: Fix config reference in LocaleMiddleware
 1.3.8: Keep query string when switching locales
 1.4.0: Add importer and exporter for messages
+1.4.1: !!! OctoberCMS Build 444 is needed in order to use this release (ref. dynamic property fix).
```

Comment:
Please change to `!!! Please update OctoberCMS to Build 444 before updating this plugin. Added ability to translate RainLab.Pages static pages`